stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -335,6 +335,7 @@ internal class ChatDomainImpl internal constructor(
         setChannelUnreadCount(me.unreadChannels)
 
         setBanned(me.banned)
+        getActiveQueries().forEach { queryChannelsController -> queryChannelsController.updateMutedChannels(me.channelMutes) }
```

Comment:
Actually in future we should get rid off it since we have all data to calculate mutes in `currentUser`. We make doubling storing it in `QueryChannelsController`
I created [Jira task](https://stream-io.atlassian.net/browse/CAS-1299) about this refactoring